### PR TITLE
docs: Update permissions for AWS

### PR DESCRIPTION
**Pull request by channingyangtm**
Update permissions for AWS

### CODE REVIEW

File: aws/required-permissions.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ Check out the templates directly for more information.
 | `logs:StopQuery` | Action | Allow Cloud One File Storage Security to stop a log query of the Lambda functions created by Cloud One File Storage Security. |
 | `logs:GetQueryResults` | Action | Allow Cloud One File Storage Security to get the log query results of the Lambda functions created by Cloud One File Storage Security. |
 | `logs:FilterLogEvents` | Action | Allow Cloud One File Storage Security to filter the log events of the Lambda functions created by Cloud One File Storage Security. |
+| `logs:TagResource` | Action | Allow Cloud One File Storage Security to tag CloudWatch log resource. |
```
Comment (jack-c-tang) [inline aws/required-permissions.md:72]: There's no role/policy that has this action in the scanner or storage stack template, so we don't need to put it here.

**Review by jack-c-tang — COMMENTED** [review]
Please use `docs:` instead of `doc:` for the commit message.

File: aws/required-permissions.md (hunk 1 above)
Comment (channingyangtm) [reply aws/required-permissions.md]: Removed

File: aws/deployment-role-policy.json [hunk 2]
```diff
@@ -31,7 +31,8 @@
         "logs:DescribeLogGroups",
         "logs:DeleteLogGroup",
         "logs:PutRetentionPolicy",
-        "logs:CreateLogGroup"
+        "logs:CreateLogGroup",
+        "logs:TagResource"
```
Comment (jack-c-tang) [inline aws/deployment-role-policy.json:35]: How about `logs:UntagResource`?

Comment (channingyangtm) [reply aws/deployment-role-policy.json:35]: `logs:UntagResource` we not used in policy.json([url](https://dsgithub.trendmicro.com/file-storage-security/cloudformation-templates/pull/166/files)), i think that is not required .

**Pull request merged**